frontend/public/components/monitoring/dashboards/index.tsx

```diff
@@ -218,38 +225,54 @@ const VariableDropdown: React.FC<VariableDropdownProps> = ({ id, name, namespace
 
   const [isError, setIsError] = React.useState(false);
 
+  const dataSourceName = variable?.datasource?.name;
+  const extensions = useExtensions<DataSourceExtension>(isDataSource);
+  const extension = extensions.find((ext) => ext.pluginName === 'dashboards-datasource-plugin');
```

Comment:
This would mean that **only** our plugin could provide custom datasources. We should not filter using the plugin name but use the `contextId` 
```suggestion
  const extension = extensions.find((ext) => ext.contextId === 'monitoring-dashboards');
```